config/main.py

```diff
@@ -753,23 +753,23 @@ def _restart_services(config_db):
 
 def interface_is_in_vlan(vlan_member_table, interface_name):
     """ Check if an interface  is in a vlan """
-    for _,intf in vlan_member_table.keys():
+    for _,intf in list(vlan_member_table.keys()):
```

Comment:
>list [](start = 18, length = 4)

The same

Reply:
This change was made by the `2to3` tool (see [here](https://www.python.org/dev/peps/pep-0469/#migrating-directly-to-python-3)):

> Note that in many cases d.keys() can be replaced by just d, but the 2to3 migration tool doesn't attempt that replacement.

Plus, keeping `.keys()` makes it clear what the intention is.

Reply:
However, I did add a space after the comma.